config/prod/wpoehlm-parallel-ec2-scicomp.yaml

```diff
@@ -0,0 +1,28 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v6.yaml
+stack_name: wpoehlm-parallel-ec2
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "NDR"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "william.poehlman@sagebase.org"
+
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "c5.18xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "350"
+
+  # Integration with our jumpcloud directory service (do not change)
+  JcConnectKey: !ssm /infra/JcConnectKey
+  JcServiceApiKey: !ssm /infra/JcServiceApiKey
+  JcSystemsGroupId: !ssm /infra/JcSystemsGroupId
+
+# For CI system (do not change)
+hooks:
+  after_create:
```

Comment:
missing a few hooks. please get all hooks from v7 yaml

Reply:
@zaro0508 didn't see a v7.yaml file in the templates folder, is this the correct way to refer to the new template?

Reply:
yes